lib/nimble_template/helpers/credo.ex

```diff
@@ -0,0 +1,10 @@
+defmodule NimbleTemplate.CredoHelper do
+  alias NimbleTemplate.Generator
+
+  @spec disable_rule(String.t(), String.t()) :: :ok | {:error, :failed_to_read_file}
+  def disable_rule(file_path, rule) do
```

Comment:
do we need to check the existence of the input `rule` @hanam1ni 🤔

Reply:
@nvminhtue IMO, we don't need to check because if a `rule` is not given, only single param is passed to this function and it will raise `FunctionClauseError`. And if an empty string is passed here, the function can work properly and doesn't break the app.